tests/integration/wapiti/templates_and_data.py

```diff
@@ -76,23 +76,44 @@
                                             'referer': '',
                                             'wstg': []}],
+                        'Clickjacking Protection': [
+                            {
+                                'curl_command': '',
+                                'detail': {
+                                    'response': {
+                                        'body': '',
+                                        'headers': [],
+                                        'status_code': 0
+                                    }
+                                },
+                                'http_request': '',
+                                'info': '',
+                                'level': 0,
+                                'method': '',
+                                'module': '',
+                                'parameter': '',
+                                'path': '',
+                                'referer': '',
+                                'wstg': []
+                            }
+                        ],
                         'Command execution': [{'curl_command': '',
-                                              'detail': {
-                                                  'response': {
-                                                      'body': '',
-                                                      'headers': [],
-                                                      'status_code': 0
-                                                  }
-                                              },
-                            'http_request': '',
-                            'info': '',
-                            'level': 0,
-                            'method': '',
-                            'module': '',
-                            'parameter': '',
-                            'path': '',
-                            'referer': '',
-                            'wstg': []}],
+                                               'detail': {
+                                                   'response': {
+                                                       'body': '',
+                                                       'headers': [],
+                                                       'status_code': 0
+                                                   }
+                                               },
+                                               'http_request': '',
+                                               'info': '',
+                                               'level': 0,
+                                               'method': '',
+                                               'module': '',
+                                               'parameter': '',
+                                               'path': '',
+                                               'referer': '',
+                                               'wstg': []}],
```

Comment:
well, since indentation is done for clickjacking, HSTS, Mime type and Unencrypted channels..

Reply:
changed :)